Tools/Scripts/libraries/webkitcorepy/webkitcorepy/string_utils.py

```diff
@@ -76,6 +76,16 @@ def join(list, conjunction='and'):
     return '{} {} {}'.format(', '.join(list[:-1]), conjunction, list[-1])
 
 
+def split(string, conjunction='and'):
```

Comment:
We could, although even if we did, the function is fairly brittle as written and needs more work.